services/inputhost/exthost.go

```diff
@@ -304,8 +319,10 @@ func (conn *extHost) close() {
 	conn.lk.Unlock() // no longer need the lock
 
 	conn.logger.WithFields(bark.Fields{
-		`sentSeqNo`: conn.seqNo,
-		`ackSeqNo`:  conn.lastSuccessSeqNo,
+		`sentSeqNo`:      conn.seqNo,
+		`ackSeqNo`:       conn.lastSuccessSeqNo,
+		`currentSizeMiB`: conn.getSizeInMiB(conn.currSizeBytes),
```

Comment:
just print the currSizeBytes and maxSizeBytes as such, to avoid any rounding and/or precision errors while conversion.  this can be valuable info for debugging.